### PR TITLE
WIP: make JavaSoft Oracle Registry Keys Optional 

### PR DESCRIPTION
**Pull request by junyuan-z**

### CODE REVIEW

File: wix/Build.OpenJDK_generic.cmd [hunk 1]
```diff
@@ -24,6 +25,12 @@ IF NOT DEFINED MSI_PRODUCT_VERSION SET ERR=6
 IF NOT DEFINED ARCH SET ERR=7
 IF NOT DEFINED JVM SET ERR=8
 IF NOT DEFINED PRODUCT_CATEGORY SET ERR=9
+IF NOT DEFINED ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION (
+    ECHO ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION is set to true
+    SET ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION=true
+) ELSE (
+    ECHO ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION is set to false
```
[the remark this answers is not in the document]
Comment (douph1) [reply wix/Build.OpenJDK_generic.cmd:32]: maybe better like this ?

<pre>
IF NOT DEFINED ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION (
    ECHO ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION is set to (default) true
    SET ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION=true
) ELSE (
    IF "%ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION%" == "false" (
        ECHO ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION is set to false
       ) ELSE (
          ECHO ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION is set to true
          SET ENABLE_JAVASOFT_ORACLE_REGISTRY_KEYS_OPTION=true
       )
)
</pre>

**Comment by karianna** [comment]
@junyuan-z can you rebase?

**Comment by douph1** [comment]
Did you always need it ? I don't see "Oracle Reg key" option on Microsoft OpenJDK build.
I suppose you reuse the installer project. Have you forked it ?

**Comment by junyuan-z** [comment]
@douph1 yes, we have an internal repo. i will work on this later this week to complete this PR.

**Comment by gdams** [comment]
Closing this one for now

**Pull request closed without merging**